### ISSUE TITLE
[digitizer] tableFields NULL values in column are displayed as null instead empty

### ISSUE THREAD

**Issue by astroidex** [mapbender/mapbender#1356]
**Describe the bug**
You can add columns of your table to the tableFields definition to show them in the Table. You could add a field with NULL values. NULL values are displayed with null instead of an empty value.

**To Reproduce**
1. Configure a digitizer
3.  add a column with Null values to the tableFields-definition
5.  See error: null will be displayed in the table

```
  tableFields:
    gid:
      label: Nr.
      width: 20%
    type:
      label: 'Typ:'
      width: 20%
    name:
      label: Name
      width: 60%
```
![Bildschirmfoto vom 2021-05-03 08-44-10](https://user-images.githubusercontent.com/996298/116848173-544e9600-abec-11eb-9276-cdc866eba786.png)

**Expected behavior**
Show an empty column instead of null

**Versions (please complete the following information if it helps to narrow down the bug):**
 - Mapbender: 3.2.4 with digitizer 1.4.7

**Comment by rschwammborn** [comment]
Hello @astroidex ,
I could reproduce this issue, so I created a ticket for the developers.

**Comment by werrolf** [comment]
See [data manager 1.2.1](https://github.com/mapbender/data-manager/releases/tag/1.2.1).
Included in Mapbender Starter v3.2.6
Manual update in existing project:
```sh
bin/composer update mapbender/data-manager
```
